Casks/apache-couchdb.rb

```diff
@@ -1,27 +1,23 @@
 cask "apache-couchdb" do
-  version "3.2.2"
-  sha256 "985d15aad597699e61c3654597ede37e43b9e13062c3b7e1d69a7cd3e1e71448"
+  arch arm: "arm64", intel: "x86_64"
 
-  # 3.2.2 is the CouchDB version. This binary package
-  # 3.2.2-1 is a bugfix release with a nonstandard URL.
-  # Replace with original url in the next release.
-  # url "https://couchdbneighbourhoodie.fra1.digitaloceanspaces.com/downloads/#{version}/mac/Apache-CouchDB.zip",
-  url "https://couchdbneighbourhoodie.fra1.digitaloceanspaces.com/downloads/#{version}/mac/3.2.2-1/Apache-CouchDB.zip",
+  version "3.3.0"
+  sha256 arm:   "12cec24f5b8b068f6fdd44eb5b5c891de02547a4972a1b4ea472dda30a8ef8ca",
+         intel: "02b14f431cdcf44b4d0c83d49c47932536f6600a9406ba165cf7a9ac83f4317c"
+
+  url "https://couchdbneighbourhoodie.fra1.digitaloceanspaces.com/downloads/#{version}/mac/#{arch}/Apache-CouchDB.zip",
       verified: "couchdbneighbourhoodie.fra1.digitaloceanspaces.com/"
   name "Apache CouchDB"
   desc "Multi-master syncing database"
   homepage "https://couchdb.apache.org/"
 
   livecheck do
     url "https://neighbourhood.ie/download-apache-couchdb-mac/"
-    # 3.2.2 is the CouchDB version. This binary package
-    # 3.2.2-1 is a bugfix release with a nonstandard URL.
-    # Replace with original regex in the next release.
-    # regex(%r{href=.*?/(\d+(?:\.\d+)+)/mac/Apache[._-]?CouchDB\.zip}i)
-    regex(%r{href=.*?/(\d+(?:\.\d+)+)/mac/3.2.2-1/Apache[._-]?CouchDB\.zip}i)
+    strategy :page_match
+    regex(%r{href=.*?/(\d+(?:\.\d+)*)/mac/(arm64|x86_64)/Apache-CouchDB\.zip}i)
   end
 
-  depends_on macos: ">= :mojave"
+  depends_on macos: ">= :ventura"
```

Comment:
Is this documented anywhere? That seems like an unreasonably big jump.

Reply:
I’m building these and I’m not ready to support older versions at this point.

Reply:
Is it in the documentation somewhere? Would be good to have a reference for this because people are going to complain when this gets merged.

Reply:
yes, the original download site mentions this: https://neighbourhood.ie/download-apache-couchdb-mac/

Reply:
Let's add it here then
```suggestion
  # see https://neighbourhood.ie/download-apache-couchdb-mac/
  depends_on macos: ">= :ventura"
```

Reply:
would be great if the style checker catches this, then :) ta

Reply:
Well, it's not so much a style issue. It's more of a "that's unexpected".

Reply:
> Well, it's not so much a style issue.

sorry that was meant for the other thread, fixed now :)